array-stack-queue/5b.c

```diff
@@ -0,0 +1,79 @@
+#define MAX 50
+#include<stdio.h>
+#include<ctype.h>
+#include<math.h>
+#include<string.h>
+
+int push(int);
+int pop();
+
+int top=-1,i;
+char a[MAX],stack[MAX];
```

Comment:
Why do you put the string variable ```a``` outsite of the main-function? You can this put right into the main-function